### PR TITLE
Use mongo:3.2, sync docker-compose.yml

### PR DESCRIPTION
**Pull request by warrenvw**
docker-compose.circle.yml -> docker.compose.yml

### CODE REVIEW

File: docker-compose.override.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ rabbitmq:
     - "5672:5672"
 
 mongodb:
-  image: mongo
+  image: mongo:3.2
```
Comment (armab) [inline docker-compose.override.yml]: :+1:

File: docker-compose.yml [hunk 2]
```diff
@@ -152,18 +152,18 @@ xenialtest:
...
 
 centos6test:
-  image: quay.io/dennybaa/droneunit:centos6-sshd
+  image: stackstorm/packagingtest:centos6
   extends:
     file: docker-compose.override.yml
     service: volumes-compose
 
 centos7test:
-  image: quay.io/dennybaa/droneunit:centos7-systemd
```
Comment (armab) [inline docker-compose.yml]: +1

**Comment by armab** [comment]
The change looks good :+1: 

@warrenvw just don't forget to provide a bit more detailed PR description so others could understand what's going on.

In short, what we do in this PR:
- Fix `docker-compose` for local builds
  - Pin `mongo` Docker to `3.2` version (`latest` mongo Docker breaks StackStorm due to https://github.com/StackStorm/st2/issues/3100)
  - Sync-up `docker-compose.yml` (local build) with `docker-compose.circle.yml` (CircleCI build), use the same Docker images.

**Comment by armab** [comment]
cc @enykeev, so when you plan to migrate to `quay.io` for build/test images, don't forget to update all `docker-compose` files as well.

File: docker-compose.override.yml (hunk 1 above)
Comment (Kami) [inline docker-compose.override.yml]: Good catch, indeed.

**Pull request merged**